recipes/flax/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "flax" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e995c4a40d4790751fbe08a44d7afec6fb22ac3e88a5f7912c40130cef81a3fa
+
+build:
+  skip: true  # [win or py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - dataclasses
+    - jax >=0.2.6
+    - matplotlib
```

Comment:
Unless you need Qt for use with matplotlib, best to depend on `matplotlib-base`.

Reply:
fixed.